code/game/machinery/rechargestation.dm

```diff
@@ -295,4 +307,30 @@
 		else
 			to_chat(user, SPAN_DANGER("Cancelled loading [C] into the charger. You and [C] must stay still!"))
 		return
+	
+	else if(istype(C, /mob/living/carbon/human/machine)) // IPCs don't take as long
```

Comment:
```suggestion
	else if(isipc(C)) // IPCs don't take as long
```

This helper would probably be better here.

Reply:
Done!